src/components/core/Markdown/List.tsx

```diff
@@ -0,0 +1,123 @@
+import IconContainer from './IconContainer';
+import { useCallback } from 'react';
+
+export default function Bold({
+  selectedRange,
+  textAreaRef,
+  setSelectedRange,
+  setUserDragged,
+}: {
+  selectedRange: number[];
+  textAreaRef: React.RefObject<HTMLTextAreaElement>;
+  setSelectedRange: (value: number[] | ((prevVar: number[]) => number[])) => void;
+  setUserDragged: (value: boolean) => void;
+}) {
+  const handleClick = useCallback(() => {
```

Comment:
@kaitoo1 I agree this would be a good candidate for testing. Happy to talk through next steps & the best way to do that

Reply:
+1, could look something like this:
1. render a text area that renders this List component (or bold, etc.)
2. simulate interactions with the component
3. test the result

found this resource: https://testing-library.com/docs/ecosystem-user-event/#typeelement-text-options

![image](https://user-images.githubusercontent.com/12162433/168400168-7da44dd5-7341-4ba0-aabe-ca96b9386bb0.png)

Reply:
overall i don't think testing needs to block this PR if we have a good sense of it working for most cases.

@connorrothschild wanna give it a whirl on your own by going through the docs, and afterwards we can pair some time next week?

in our codebase i would:
1. make a test file next to the component file (e.g. `Bold.test.tsx` next to `Bold.tsx`)
2. define a test. you can use the format we use in other tests, like [this one](https://github.com/gallery-so/gallery/blob/main/src/contexts/collectionEditor/deduplicateObjectByOpenseaIdAndPreferEarliest.test.ts)
3. to run just the test you defined in isolation, you would do: `yarn jest <name of test>`. so if your test suite is `describe('Bold', () => ...)` you should be able to run `yarn jest Bold`



yarn jest deduplicateObjectByOpenseaIdAndPreferEarliest

Reply:
oh also make sure to name the file with `.test.tsx` instead of `.test.ts`, bc it'll be rendering JSX

Reply:
thinking about this further- i actually think you just need to write one overarching test for `MarkdownShortcuts.test.tsx`, since it encompasses all the different kinds of editing (instead of 3 different test files). this is more integration-y and preferred